### PR TITLE
DOC: Plans for 2.7

### PR DESCRIPTION
**Pull request by TomAugspurger**
Closes https://github.com/pandas-dev/pandas/issues/18894

Just some sample text, essentially following NumPy.

Of course, it'd be nice if 1.0 happened to be the LTS, but we'll see.

### CODE REVIEW

**Comment by jorisvandenbossche** [comment]
Do we need to send a notice about this to the pydata/pandas-dev mailing list to give people the chance to give feedback?

**Comment by jreback** [comment]
lgtm

this i already promise enough (maybe even too much)

merge away

File: doc/source/install.rst [hunk 1]
```diff
@@ -15,6 +15,31 @@ Instructions for installing from source,
...
+
+The final major release before **December 31, 2018** will be a long term support (LTS) release.
+The pandas maintainers will backport bugfixes and make releases supporting Python 2 to that branch.
+For example, if ``0.25.0`` is the last major release, we'll make make releases for ``0.25.1``, ``0.25.2``, etc., as needed.
+This branch will not receive new features.
+
+Starting **January 1, 2019**, all major release will be Python 3 only.
+
+Starting **January 1, 2020**, the pandas maintainers will stop supporting Python 2. The released packages will still be
+available on PyPI and through conda, but will not recieve any more bugfixes.
```
Comment (jschendel) [inline doc/source/install.rst]: minor typo: recieve --> receive

**Comment by TomAugspurger** [comment]
> Do we need to send a notice about this to the pydata/pandas-dev mailing list to give people the chance to give feedback?

Sure, are we comfortable with the proposal and timeline? Personally, I'm not sure how much backporting we can commit to doing over 2019. Is there any benefit to being ahead of NumPy here, by a bit, since they're a dependency?

**Comment by jreback** [comment]
actually I don't think we should commit to a LTS release at all. The final release of 2.7 is Dec 31, 2018. 
We could always add that support if someone wants to commit to it (people and funding).

**Comment by TomAugspurger** [comment]
Updated the text about 2.7 support after December 31, 2018 to reflect the default of "we're done with 2.7. If someone wants to continue supporting it, let us know."

**Comment by jreback** [comment]
lgtm. @jorisvandenbossche

**Comment by TomAugspurger** [comment]
I'll send out a note to the mailing list.

**Review by auvipy — COMMENTED** [review]
that sounds reasonable time line

**Comment by jreback** [comment]
thanks @TomAugspurger

**Comment by topper-123** [comment]
The link in the whatsnew warning doesn't work: http://pandas-docs.github.io/pandas-docs-travis/whatsnew.html

**Comment by hugovk** [comment]
I've created a PR to http://www.python3statement.org to add pandas to the timeline:

https://github.com/python3statement/python3statement.github.io/pull/122

**Pull request merged**